modules/ROOT/pages/metrics-archive-api.adoc

```diff
@@ -0,0 +1,33 @@
+= Anypoint Monitoring Archive API
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Anypoint Monitoring Archive API enables you to download metric data for APIs. The data that this API archives appears in your xref:monitoring::api-analytics-dashboard.adoc#enhanced-api-metrics[enhanced API metrics dashboards]. However, the data contained in the archive is asynchronous.
```

Comment:
Should we document the supported Mule runtime versions?  Not all versions are supported.